reposerver/cache/cache.go

```diff
@@ -39,8 +44,8 @@ type ClusterRuntimeInfo interface {
 	GetKubeVersion() string
 }
 
-func NewCache(cache *cacheutil.Cache, repoCacheExpiration time.Duration, revisionCacheExpiration time.Duration) *Cache {
-	return &Cache{cache, repoCacheExpiration, revisionCacheExpiration}
+func NewCache(cache *cacheutil.Cache, repoCacheExpiration time.Duration, revisionCacheExpiration time.Duration, revisionCacheLockTimeout time.Duration) *Cache {
```

Comment:
Okay, yeah I can do that, when the original PR was discussed it was said to hard code instead of use the variables like I had originally, but I can go back to that configuration.

Reply:
Swapped this back and re-added test for disabling when the time is set to 0